Doc/library/argparse.rst

```diff
@@ -1852,10 +1852,13 @@ Sub-commands
      >>> parser.parse_args(['co', 'bar'])
      Namespace(foo='bar')
 
-   One particularly effective way of handling sub-commands is to combine the use
-   of the :meth:`add_subparsers` method with calls to :meth:`set_defaults` so
-   that each subparser knows which Python function it should execute.  For
-   example::
+   One particularly effective way of handling sub-commands is to
+   combine the use of the :meth:`add_subparsers` method with calls to
+   :meth:`set_defaults` so that each subparser knows which Python
+   function it should execute.  The :meth:`set_defaults` method of the
```

Comment:
I'd recommend keeping the original structure so just the new changes can be more easily reviewed.

```suggestion
   One particularly effective way of handling sub-commands is to combine the use
   of the :meth:`add_subparsers` method with calls to :meth:`set_defaults` so
   that each subparser knows which Python function it should execute.
   The :meth:`set_defaults` method of the
```